api-test/src/test/java/org/openmrs/module/metadatasharing/util/MetadataUtilTest.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.metadatasharing.util;
+
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class MetadataUtilTest extends BaseModuleContextSensitiveTest {
+	
+	@Test
+	public void testSetupStandardMetadata() throws Exception {
+        assertThat(Context.getVisitService().getVisitTypeByUuid("86b3d7bc-d91f-4ce2-991c-f71bba0b31e4"), nullValue());
+
+        boolean anyChanges = MetadataUtil.setupStandardMetadata(getClass().getClassLoader());
+
+        assertTrue(anyChanges);
+        assertThat(Context.getVisitService().getVisitTypeByUuid("86b3d7bc-d91f-4ce2-991c-f71bba0b31e4").getName(), is("Clinic or Hospital Visit"));
+	}

```

Comment:
and here.

Reply:
@HelioStrike  the Indention has no problem because the class is more alligned to the left than the mothods. and the methods are more alligned to the left than the statements. i dont think theres any indention problem

Reply:
The tabs didn't seem to align well so I pointed it out even though the ordering is correct, as I wasn't sure it was acceptable. Sorry about that.

Reply:
Indentation is definitely wrong. Can you run an autoformatter with the OpenMRS code style?

Reply:
i ran the auto formatter